README.md

```diff
@@ -147,9 +147,9 @@ BSD 3-Clause
 
 ### References
 
-Many of the engineering and encoding functionality is inspired by this [series of articles from the 2009 KDD competition](http://www.mtome.com/Publications/CiML/CiML-v3-book.pdf).
+Many of the engineering and encoding functionalities are inspired from this [series of articles from the 2009 KDD Competition](http://www.mtome.com/Publications/CiML/CiML-v3-book.pdf).
```

Comment:
I think inspired should be by instead of from. Could we keep the original?

Reply:
You're right! Actually both are grammatically okay, but 'by' would be more suitable in this context.